### PR TITLE
Update BOTMETA.yml

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Update BOTMETA.yml

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`.github/BOTMETA.yml`

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -113,6 +119,10 @@ files:
   $lookups/dig.py:
     maintainers: jpmens
     labels: dig
+  $lookups/dss.py:
```
Comment (felixfontein) [inline .github/BOTMETA.yml:122]:
```suggestion
  $lookups/tss.py:
```

Comment (Andersson007) [reply .github/BOTMETA.yml:122]: ops, thanks!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks for updating BOTMETA!

**Comment by Andersson007** [comment]
@felixfontein thanks!

**Pull request merged**